kubernetes/e2e_test/base.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import urllib3
+
+
+def _is_k8s_running():
```

Comment:
nit: _ at the start used to hide methods in classes. Here the method is being used in this module and it is not exported in its __init__.py. Thus I don't see any reason to add _.